src/lib/components/event/event-shortcut-keys.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+  import Shortcut from '$lib/holocene/keyboard-shortcut/shortcut.svelte';
+  export let open = false;
+  export let compact: boolean;
+</script>
+
+{#if open}
+  <div class="modal">
```

Comment:
Why not use holocene's modal?

Reply:
Because the Modal event listeners get overridden by the api-pagination event listeners. (i.e. Close / x doesn't work). Also, doesn't need button. But also... waiting on Ash's designs for this so it will change. Just wanted something out there for now. 

Not sure if 'Space' will be the key to trigger open/close either.

Reply:
Hmm I see. Maybe we do away with `Space`  and just have an Icon button next to the pagination controls?

<img width="196" alt="Screen Shot 2022-12-15 at 11 57 40 AM" src="https://user-images.githubusercontent.com/11775628/207944731-e6b81b5c-7992-4edd-849e-e24e90464fdd.png">

As of right now, you'd have to know the 'Space' keyboard shortcut to see the pagination keyboard shortcuts

Reply:
Yeah for sure need a prompt or something to tell them how to view it. I'll ask Ash what they have in mind.